conf.py

```diff
@@ -98,6 +99,24 @@
 #
 html_theme = 'sphinx_rtd_theme'
 
+html_title = 'EPICS Documentation'
+html_short_title = 'EPICS Docs'
+html_baseurl = 'https://docs.epics-controls.org/'
```

Comment:
ReadTheDocs actually recommends against setting `html_baseurl`: https://rtd-with-docsearch.readthedocs.io/en/latest/guides/canonical-urls.html#sphinx

Or using the [`READTHEDOCS_CANONICAL_URL` env var](https://docs.readthedocs.com/platform/stable/intro/sphinx.html#set-the-canonical-url)

```python
html_baseurl = os.environ.get("READTHEDOCS_CANONICAL_URL", "/")
```

This is because the root URL of the documentation won't be https://docs.epics-controls.org/, but `https://docs.epics-controls.org/{lang}/{version}`

You can see this in the RTD output: https://epics--127.org.readthedocs.build/en/127/sitemap.xml the links are all invalid because it assumes the documentation is at the root of the domain.